src/screensComponents/modal/index.js

```diff
@@ -1,13 +1,12 @@
 import _ from 'lodash';
```

Comment:
a general comment on this whole issue..
I noticed that on a non-RNN project (our expo app for that matter) there's no issue. 
If you'll run the expo app on Android and open a dialog you will see that it covers the whole screen. 

I think that's the reason it was all connected to RNN. 
It's possible we need to find a way to fix it specifically for RNN.

Reply:
If it only happens when RNN is there, then I think we should close this PR, and wait for RN's update - see our Jira ticket (595), or maybe open a ticket fir RNN.

Reply:
I think the recent feature of RNN, that allows on Android to `drawBehind` status bar is the solution
Can u try setting that and see if it works. 
If it does we need to figure how to make the Modal set it automatically.

Reply:
Isn't the `drawBehind` for the RNN's navigation bar? - I think Guy said it's a known bug, and we tried to find a workaround for it.